### PR TITLE
Disable hanging tests

### PR DESCRIPTION
**Pull request by Forgind**
NormalPriorityBuild has hung several times. I haven't been able to figure out why it would hang and other tests that use the same path (ExecuteMSBuildExeExpectSuccess) wouldn't.

### CODE REVIEW

File: src/MSBuild.UnitTests/XMake_Tests.cs [hunk 1]
```diff
@@ -1250,7 +1251,6 @@ private void RunPriorityBuildTest(ProcessPriorityClass expectedPrority, params s
             IDictionary<string, string> environmentVars = new Dictionary<string, string>
             {
                 { "MSBUILDNOINPROCNODE", "1"},
-                { "DISABLECONSOLECOLOR", "1"},
```
Comment (rainersigwald) [inline src/MSBuild.UnitTests/XMake_Tests.cs:1253]: Why this change?

File: src/MSBuild.UnitTests/XMake_Tests.cs [hunk 2]
```diff
@@ -1212,6 +1212,7 @@ public void ResponseFileSupportsThisFileDirectory()
             }
         }
 
+        /* These tests hung repeatedly. See https://github.com/microsoft/msbuild/issues/5368
```
Comment (rainersigwald) [inline src/MSBuild.UnitTests/XMake_Tests.cs:1215]: Don't comment tests; instead, use 

```C#
[Fact(Skip = "url of bug")]
```

File: src/MSBuild.UnitTests/XMake_Tests.cs [hunk 3]
```diff
@@ -1259,8 +1259,9 @@ private void RunPriorityBuildTest(ProcessPriorityClass expectedPrority, params s
             string expected = string.Format(@"Task priority is '{0}'", expectedPrority);
             logContents.ShouldContain(expected, () => logContents);
         }
+        */
 
-#region IgnoreProjectExtensionTests
+        #region IgnoreProjectExtensionTests
```
Comment (rainersigwald) [inline src/MSBuild.UnitTests/XMake_Tests.cs:1264]: Revert whitespace change please.

File: src/MSBuild.UnitTests/XMake_Tests.cs (hunk 1 above)
Comment (Forgind) [reply src/MSBuild.UnitTests/XMake_Tests.cs:1253]: This was part of trying to figure out why it was hanging, but I forgot to revert it. Will fix.
